src/features/mcda/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { toolbarControlsAtom } from '~core/shared_state';
+import { controlGroup, controlVisualGroup } from '~core/shared_state/toolbarControls';
+import { mcdaCalculationAtom } from './atoms/mcdaCalculation';
+import type { JsonMCDA } from './atoms/mcdaCalculation';
+
+const example = `
+{
+  "id": "MCDA",
+  "axes":[
+    ["population", "area_km2"],
+    ["hazardous_days_count", "one"],
+    ["highway_length", "total_road_length"]
+  ],
+  "ranges":[
+    [0, 46200],
+    [0, 365],
+    [0, 1]
+  ],
+  "sentiments":[
+    ["good", "bad"],
+    ["good", "bad"],
+    ["bad", "good"]
+  ],
+  "coefficients":[
+    1,
+    1,
+    1
+  ],
+  "colors": {
+    "good": "rgba(90, 200, 127, 0.5)",
+    "bad": "rgba(228, 26, 28, 0.5)"
+  }
+}
+`;
+
+export function initMCDA() {
+  toolbarControlsAtom.addControl.dispatch({
+    id: 'MCDA',
+    name: 'MCDA',
+    title: 'MCDA',
+    active: false,
+    visualGroup: controlVisualGroup.noAnalytics,
+    exclusiveGroup: controlGroup.mapTools,
+    icon: (
+      <div style={{ height: '24px', display: 'flex', alignItems: 'center' }}>MCDA</div>
+    ),
+    onClick: () => {
+      const jsonString = prompt(
```

Comment:
need this backlogged to be able to do in our components library too

Reply:
we have a panel, that can be used as a modal
we can extend current implementation in next task